### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by gerroon** [nikorummukainen/blender-python-code-templates#3]
Hi

Can you please add documentation for how to use this in Vscode.

thanks

**Comment by ChristianVaughn** [comment]
Anyone figure out how to use this? I installed it and dont see anything in the command map relating to the word blender

**Comment by alanscodelog** [comment]
They are snippets. Just start typing one of the prefixes in the readme (the left column) and you should get auto-suggested the snippet. Not sure if the extension is still working because it hasn't been updated in a while, but that's how it worked when I last used it.

**Comment by ChristianVaughn** [comment]
I figured it out. It still works, but the templates are not up to date with
the most recent version of the blender api

**Comment by alanscodelog** [comment]
Yeah, there's also that, I forgot. It's been a while since I've done any Blender stuff. 

For those who stumbled onto this extension who wanted to use this because they just started learning the api, I think most (not sure about the small snippets) are in the Blender text editor (in the top bar under Templates).

The author wrote a snippet generator addon to extract them out but it probably doesn't work now either and unfortunately I don't really have the time to fix any of this, but I do have write/publish access to this repo/extension so if anyone is interested enough to at least make a branch that updates the snippet generator (I do not have access to the snippet generator repo), I can update this addon.
